src/main/java/bio/terra/cda/app/generators/SqlGenerator.java

```diff
@@ -140,11 +142,28 @@ protected QueryContext buildQueryContext(
         .setViewListBuilder(viewListBuilder);
   }
 
-  public QueryJobConfiguration.Builder generate() throws IllegalArgumentException {
+  private void buildLabels(Query query, Map<String, String> queryMap) {
+    if (Objects.isNull(query)) {
+      return;
+    }
+    if (query.getNodeType().equals(Query.NodeTypeEnum.OFFSET)) {
+      queryMap.put("query_offset", query.getValue());
+    }
+    if (query.getNodeType().equals(Query.NodeTypeEnum.LIMIT)) {
+      queryMap.put("query_limit", query.getValue());
+    }
+    buildLabels(query.getL(), queryMap);
+    buildLabels(query.getR(), queryMap);
+  }
+
+  public QueryJobConfiguration.Builder generate()
+      throws IllegalArgumentException, JsonProcessingException {
     String querySql = sql(qualifiedTable, rootQuery, false, false, false);
     QueryJobConfiguration.Builder queryJobConfigBuilder =
         QueryJobConfiguration.newBuilder(querySql);
-
+    Map<String, String> qMap = new HashMap<>();
```

Comment:
These are no longer necessary because we are not using the labels anymore.

Reply:
updated and fixed